### PR TITLE
Implement project_review and review_rating join tables

### PR DESCRIPTION
**Pull request by hnlee**

### CODE REVIEW

File: spec/factories/review_ratings.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :review_rating do
+    review nil
+rating nil
```
Comment (NicoleCarpenter) [inline spec/factories/review_ratings.rb]: Can you please fix this spacing?

File: spec/factories/project_reviews.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :project_review do
+    project nil
+review nil
```
Comment (NicoleCarpenter) [inline spec/factories/project_reviews.rb]: Can you please fix this spacing?

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
-    review = Review.new(review_params)
-    review.project = project
+    review = Review.new(content: review_params[:content])
     if review.save
-      redirect_to project
-    else
-      redirect_to new_project_review_path(:review => params[:review])
+     project_review = ProjectReview.create(project_id: review_params[:project_id],
+                                            review_id: review.id)
+      redirect_to project_path(review_params[:project_id])
```
Comment (NicoleCarpenter) [inline app/controllers/reviews_controller.rb]: Can you please fix this spacing?

File: spec/factories/project_reviews.rb (hunk 2 above)
Comment (NicoleCarpenter) [inline spec/factories/project_reviews.rb]: Can you please fix this spacing?

File: spec/factories/review_ratings.rb (hunk 1 above)
Comment (NicoleCarpenter) [inline spec/factories/review_ratings.rb]: Can you please fix this spacing?

File: app/controllers/reviews_controller.rb (hunk 3 above)
Comment (NicoleCarpenter) [inline app/controllers/reviews_controller.rb]: Can you please fix this spacing?

File: spec/integration/project_show_spec.rb [hunk 4]
```diff
@@ -1,30 +1,39 @@
 require 'spec_helper'
 
-describe 'review' do
+describe 'review', :type => :feature do
```
Comment (NicoleCarpenter) [inline spec/integration/project_show_spec.rb]: What does this do? I have not seen this syntax.

**Comment by hnlee** [comment]
@NicoleCarpenter Fixed spacing as requested!

**Comment by NicoleCarpenter** [comment]
👍

**Pull request merged**